setup-files/gerald-pr.yml

```diff
@@ -1,21 +1,17 @@
 name: Gerald - Notify and Request Reviewers On Pull Request
 'on':
- - pull_request
+  pull_request:
+    # ready_for_review isn't included by default, so we add it
+    # here to have this workflow re-run when a pr is converted
+    # from "draft" to "not draft".
+    # We also add "edited" to re-check when the base branch
+    # is changed.
+    types: [opened, synchronize, reopened, ready_for_review, edited]
```

Comment:
ASIDE: I wish edited could be "base branch changed" and then title edits wouldn't cause a rerun. I don't think they have a trigger that specific though. No change requested.

Reply:
yup yup.
looks like we ought to be able to determine that it was a title change and then bail though
<img width="681" alt="image" src="https://user-images.githubusercontent.com/112170/172710063-63cd9ac6-1a3c-462a-a4d1-415a5631c08e.png">

Reply:
Oh great. Something to consider for future updates.